src/components/periodic-table/PeriodicTable.test.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import PeriodicTable, { IPeriodicTableElement } from "./PeriodicTable";
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import PtElementInfo from "../pt-element/PtElementInfo";
+import ElementManager from "../../ElementManager";
+
+function elementRenderer(atomic: number): IPeriodicTableElement {
+  return {
+    // @ts-ignore fix this
+    component: PtElementInfo,
+    props: {
+      element: ElementManager.getElement(atomic),
+      onClick: undefined,
+    },
+  };
+}
+
+async function renderTable() {
+  render(<PeriodicTable elementRenderer={elementRenderer} />);
+  await waitForElementToBeRemoved(
+    () => screen.queryAllByLabelText(/loading/i),
+    { timeout: 4000 }
+  );
+}
+
+describe("should render the periodic table", () => {
+  beforeAll(() => {
+    ElementManager.loadElements();
+  });
+
+  test("should show a spinner", async () => {
+    const { container } = render(
```

Comment:
It looks like we're not using `container` , can we remove it?

Reply:
Yes